test/unit/BPool/BPool_SwapExactAmountIn.t.sol

```diff
@@ -31,11 +31,10 @@ contract BPoolSwapExactAmountIn is BPoolBase, BNum {
 
   function setUp() public virtual override {
     super.setUp();
+    tokenIn = tokens[0];
+    tokenOut = tokens[1];
```

Comment:
i like this, tho seems inconsistent with the other PR, let's clean it up later

Reply:
making a list in BAL-155